core/src/main/java/org/apache/accumulo/core/fate/zookeeper/DistributedReadWriteLock.java

```diff
@@ -137,7 +137,8 @@ public void lock() {
             return;
           }
         } catch (InterruptedException ex) {
-          // ignored
+          Thread.currentThread().interrupt();
+          log.warn("Interrupted while waiting to acquire " + getClass().getSimpleName());
```

Comment:
If this were seen in the logs, I think it'd be very hard to troubleshoot, unless the user know they were the ones who interrupted it (with Ctrl-C or something). It's probably best to preserve the stack trace:

```suggestion
          log.warn("Interrupted while waiting to acquire lock", ex);
```